index.html

```diff
@@ -18,36 +18,44 @@
         <div class="icon-menu"></div>
         <ul>
           <li class="nav-item" data-content="projects"><a href="#" class="icon-github"></a>Projects</li>
-          <li class="nav-item" data-content="links"><a href="#" class="icon-terminal"></a>Links</li>
           <li class="nav-item" data-content="about"><a href="#" class="icon-barcode"></a>About</li>
+          <li class="nav-item" data-content="links"><a href="#" class="icon-terminal"></a>Links</li>
```

Comment:
Suggestion here would be to include `cursor: pointer` styles for these list items so the user expects to be able to click within the entire tab block,